### PR TITLE
Adapt the package for SUSE Manager 4.3.

### PR DESCRIPTION
**Pull request by juliogonzalez**
I also adapted the SPEC to the most recent SLE/openSUSE format guidelines.

### CODE REVIEW

File: package/skelcd-control-suse-manager-proxy.spec [hunk 1]
```diff
@@ -39,8 +39,8 @@ BuildRequires:  libxml2-tools
 BuildRequires:  yast2-installation-control >= 4.1.5
 
 # Original SLES control file (skip registration)
-BuildRequires: skelcd-control-SLES >= 15.2.0
-BuildRequires: diffutils
+BuildRequires:  diffutils
+BuildRequires:  skelcd-control-SLES >= 15.2.0
```
Comment (juliogonzalez) [inline package/skelcd-control-suse-manager-proxy.spec:43]: Do I need to change this from `15.2.0` to `15.4.0`? Even changing it the CI failed with the same message.

**Review by jreidinger — APPROVED** [review]

**Comment by juliogonzalez** [comment]
@jreidinger @lslezak can you merge this PR as well, and submit it, as you did with https://github.com/yast/skelcd-control-suse-manager-server/pull/24 ?

**Comment by juliogonzalez** [comment]
@lslezak I guess that since the bot failed, no SR was created?

**Comment by lslezak** [comment]
Fixed in #18

**Pull request merged**